src/Microsoft.Private.UnicodeData/Microsoft.Private.UnicodeData.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageVersion>1.0.0</PackageVersion>

```

Comment:
you can remove that version now, as we switch to the dependency flow system with automatic versioning.